src/pages/public_registration/DeletedPublicationInformation.tsx

```diff
@@ -0,0 +1,32 @@
+import { Registration, RegistrationStatus } from '../../types/registration.types';
+import { Box, Typography } from '@mui/material';
+import { useTranslation } from 'react-i18next';
+
+interface DeletePublicationInformationProps {
+  registration: Registration;
+}
+
+export const DeletedPublicationInformation = ({ registration }: DeletePublicationInformationProps) => {
+  const { t } = useTranslation();
+  return (
+    <>
+      {(registration.status === RegistrationStatus.Deleted ||
+        registration.status === RegistrationStatus.Unpublished) && (
+        <Box
+          aria-hidden="true" // works in pair with DeletedPublicationScreenReaderInformation
```

Comment:
Nå som denne komponenten brukes både for visning og skjermleser burde sikkert denne fjernes, og sette `aria-hidden` på tagen med `visuallyHidden` i stedet?

Tror det blir riktig ref [MDN](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Attributes/aria-hidden): 
> Adding aria-hidden="true" to an element removes that element and all of its children from the accessibility tree